script/Arcueid/Day 10/10_00_ARC10_3_1.txt

```diff
@@ -351,7 +351,7 @@ Besides, I'm bad at that game. I get sad when I play it. That's the only thing I
 [sha:a2b3aa4798ea91316c257db28dfade434ab6e92b]{
 -- Page 147, Offset 24631.
 -- 「弱いんじゃなくて苦手なんだよ。っていうかつまんないだろ、言葉を繋げても。
-"I'm not unskilled, I'm just not good at it.
+"I'm not unskilled, it's just not for me.
```

Comment:
With this change, you can change 'unskilled' to just 'bad at', at least in Arc's line, which better matches the first thing he actually says in 24629
> Besides, I'm bad at that game. [...]
> "**You're bad at it**? Is it because your vocabulary is too small?"

Reply:
I just remember this line being very silly when read in English when I saw it.
But yeah, it def needs more changes in the surroundings.

>Besides, that game just isn't for me. I get sad when I play it. It's the one thing I absolutely refuse to play.

>Wait, are you bad at it? Small vocabulary, or something?

>I'm not bad at it, it's just not for me.